### ISSUE TITLE
[WCAG] Fix Export Dialog Reflow

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/GeoView#3594]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

The Export Map icon button is currently hidden at smaller breakpoints because the Export dialog is not responsive and causes reflow issues at smaller breakpoints / large zoom levels.

### Expected Behavior

Enable the Export Map icon button at smaller breakpoints.

Ensure the Export dialog is repsonsive and can be used at smaller breakpoints / large zoom levels.

### Steps To Reproduce

1. Set browser to 1280px width
2. Zoom in to 400%
3. The dialog should be usable without horizontal scrolling or clipped UI elements.
